README.md

```diff
@@ -20,19 +13,19 @@ A work-in-progress application to simulate purchasing train tickets. This projec
 
 ### Plan and Progress
 
-Our initial plan was to translate the requirements into a set of features, required and optional, in as GitHub issues.  We started with a spreadsheet of the features and went through the provided design to determine the issues that existed with the code.  We decided the priority of the features & issues and our team members took up those issues.  Our architect created the documentation for the design including the component, sequence, and class diagrams for the system.
+Our initial plan was to translate the requirements into a set of features, required and optional, as GitHub issues.  We started with a spreadsheet of the features and went through the provided design to determine the issues that existed with the code.  We decided the priority of the features & issues and our team members took up those issues.  Our architect created the documentation for the design including the component, sequence, and class diagrams for the system.
 
 We have been able to expand on the functionality of the system, but there are still many aspects that we did not have time to complete.  We discuss the issues we encountered in the next section.
 
-- [x] Scenario 3.1.1 - Search Tickets Scenario -- added Round Trip functionality
+- [X] Scenario 3.1.1 - Search Tickets Scenario -- added Round Trip functionality
```

Comment:
looks like this is a capital X but others are lower case. Not sure if you want to change it back to match the others.

Reply:
fixed